recipe/build.sh

```diff
@@ -8,17 +8,27 @@ elif [[ ${HOST} =~ .*linux.* ]]; then
     USE_GCC=1
 fi
 
-make    \
-    USE_64=1 \
-    PREFIX="${PREFIX}" \
-    NSPR_PREFIX="${PREFIX}" \
-    NSPR_INCLUDE_DIR="${PREFIX}/include/nspr" \
-    C_INCLUDE_PATH="${PREFIX}/include:${PREFIX}/include/nspr" \
-    LIBRARY_PATH="${PREFIX}/lib" \
+make   -j1 BUILD_OPT=1 \
+    NSPR_INCLUDE_DIR=$PREFIX/include/nspr \
+    NSPR_LIB_DIR=$PREFIX/lib \
+    PREFIX=$PREFIX \
+    SQLITE_INCLUDE_DIR=$BUILD_PREFIX/include \
+    SQLITE_LIB_DIR=$BUILD_PREFIX/lib \
```

Comment:
@jakirkham , that was the only way which I found to make it pass in ``osx`` and ``linux``.

Do you have a better idea here?

Reply:
After addressing the comment below, we should be able to set these to `$PREFIX` instead.

Reply:
That will fail in one of the ``osx``, the system will not find the ``sqlite``. I will change that to you take a look